dockstore-webservice/src/main/java/io/dockstore/webservice/core/User.java

```diff
@@ -75,7 +78,7 @@
     @NamedQuery(name = "io.dockstore.webservice.core.User.findByGoogleEmail", query = "SELECT t FROM User t JOIN t.userProfiles p where( KEY(p) = 'google.com' AND p.email = :email)"),
     @NamedQuery(name = "io.dockstore.webservice.core.User.findByGitHubUsername", query = "SELECT t FROM User t JOIN t.userProfiles p where( KEY(p) = 'github.com' AND p.username = :username)") })
 @SuppressWarnings("checkstyle:magicnumber")
-public class User implements Principal, Comparable<User> {
+public class User implements Principal, Comparable<User>, Serializable {
```

Comment:
Just curious, why are you adding `Serializable` here and in other places? I know what it's for in general, but I don't think you need it for Hibernate, so I'm wondering why it's now needed.

Not arguing against; just asking.

Reply:
The way I added the organisationUser table for roles requires all of the User and Organisation objects to be serializable.

Reply:
👍